cli/xterminal/terminal.go

```diff
@@ -0,0 +1,40 @@
+//go:build !windows
+// +build !windows
+
+package xterminal
+
+import (
+	"golang.org/x/term"
+)
+
+// State differs per-platform.
+type State struct {
+	s *term.State
+}
+
+// MakeInputRaw calls term.MakeRaw on non-Windows platforms.
+func MakeInputRaw(fd uintptr) (*State, error) {
```

Comment:
It'd be nice to have one set of public functions instead of repeating the full API for both platforms.